docs/testing-deployment-publishing/publishing/apple-appstore-deployment.md

```diff
@@ -281,123 +213,32 @@ If you don't see the **Download API Key** link immediately, refresh your page.
-Please follow these steps to deploy from FlutterFlow:
-
-1. From FlutterFlow, navigate to **Settings & Integrations > App Settings >** **Mobile Deployment > App Store**.
-
-2. Scroll down to find the **App Store** section and click on the arrow icon on the right to open it.
-3. Click on the **Deploy To App Store.**
-4. A popup will appear. **Select Deploy.** If another deployment is in progress, selecting Deploy will cancel the previous deployment.
-5. **Please note: It can take a few minutes for this request to process.** Once this step is complete, your **Latest Message** will change to **Submitted.**
-6. **It can take 30 minutes to several hours for your app to be deployed to the App Store.** You can check the status of your deployment by selecting **Check Build Status.**
-7. Once complete, you will receive an email from App Store Connect that a new build has been added to your app.
-
-<div style={{
-    position: 'relative',
-    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
-    height: 0,
-    width: '100%'}}>
-    <iframe 
-        src="https://demo.arcade.software/DShxw9lEr9bqd2Ohic8r?embed&show_copy_link=true"
-        title="Sharing a Project with a User"
-        style={{
-            position: 'absolute',
-            top: 0,
-            left: 0,
-            width: '100%',
-            height: '100%',
-            colorScheme: 'light'
-        }}
-        frameborder="0"
-        loading="lazy"
-        webkitAllowFullScreen
-        mozAllowFullScreen
-        allowFullScreen
-        allow="clipboard-write">
-    </iframe>
-</div>
-
-</TabItem>
-<TabItem value="Deploy From GitHub" label="Deploy From GitHub">
+To deploy your app from FlutterFlow, go to **Settings & Integrations > App Settings > Mobile Deployment > App Store** and click **Deploy To App Store**. Once deployed, you will receive an email from App Store Connect that a new build has been added to your app.
 
-Associating the Github repo allows you to introduce the custom code into your FlutterFlow project. You may want to deploy the modified code directly from your Github repo to App Store.
+![deploy-to-appstore.avif](../imgs/deploy-to-appstore.avif)
 
-To deploy from Github repo:
+:::info
+- Every time you deploy, we'll auto increment the **Build Number** (i.e., version code in Android) to ensure that each release is identifiable. If needed, you can update the *App Version* and *Build Number* yourself.
+- If another deployment is already in progress, deploying a new build will cancel the previous one.
+- It may take a few minutes for the request to process. Once completed, the status will be updated to **Submitted**.
+:::
 
-1. If you haven't added the Github repo yet, you can do so using the instructions [here](../exporting-code/push-to-github.md).
+:::tip
+You also have the option to [**Deploy apps from your GitHub repository**](deploy-from-github.md).
+:::
```

Comment:
would suggest expanding this tip and say when would they need to deploy apps from github, like it can still be 2 sentences but something like, " if you like to manage the deployment outside of Flutterflow...." or something around, have your own CI/CD pipeline via github, manage the deployment outside of FF etc, basically give them a short reason why they need to consider this option.

Same comment for Android page. Change at both places please.

Reply:
Ya, that makes sense.